web/src/app/history/task-history-query/task-history-query.component.scss

```diff
@@ -1,12 +1,13 @@
-.example-loading-shade {
-  position: absolute;
-  top: 0;
-  left: 0;
-  bottom: 56px;
-  right: 0;
-  background: rgba(0, 0, 0, 0.15);
-  z-index: 1;
-  display: flex;
-  align-items: center;
-  justify-content: center;
+.task-history-query {
+  height: calc(100vh - 56px);
+  width: auto;
+  overflow-x: scroll;
+  overflow-y: hidden;
```

Comment:
I would set overflow-y: auto instead of hidden, so the scrollbar hides when everything is in view and displays itself when needed.

Reply:
I had trust in the pagination component 😁